pandas/core/arrays/base.py

```diff
@@ -1207,7 +1214,7 @@ def _formatter(self, boxed: bool = False) -> Callable[[Any], str | None]:
     # Reshaping
     # ------------------------------------------------------------------------
 
-    def transpose(self, *axes) -> ExtensionArray:
+    def transpose(self, *axes: tuple[int] | list[int] | None) -> ExtensionArray:
```

Comment:
do we actually do axes like this *anywhere* else?

Reply:
I actually think that the signature we have here should be changed to `axes: tuple[int] | list[int] | None` because we are doing this to be compatible with numpy.  We don't do it elsewhere, but we only use `*axes` in `DataFrame.transpose()` .